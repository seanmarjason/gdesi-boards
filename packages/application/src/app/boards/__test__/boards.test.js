import { render } from '@testing-library/react'
import Boards from '../boards';

global.fetch = jest.fn(() =>
    Promise.resolve({
      json: () => Promise.resolve(''),
    }),
  );
  
 
test('Boards renders correctly', () => {
  const { container } = render(<Boards />)
  expect(container).toMatchSnapshot()
})
