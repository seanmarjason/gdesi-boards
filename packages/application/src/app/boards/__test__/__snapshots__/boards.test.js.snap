// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Boards renders correctly 1`] = `
<div>
  <main
    class="MuiBox-root css-i9gxme"
  >
    <div
      class="MuiStack-root css-vsjeqg-MuiStack-root"
    >
      <div
        class="MuiStack-root css-1sd6chr-MuiStack-root"
      >
        <nav
          aria-label="breadcrumb"
          class="MuiTypography-root MuiTypography-body1 MuiBreadcrumbs-root css-5cvx94-MuiTypography-root-MuiBreadcrumbs-root"
        >
          <ol
            class="MuiBreadcrumbs-ol css-1uwp4ue-MuiBreadcrumbs-ol"
          >
            <li
              class="MuiBreadcrumbs-li"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 css-1d2i4tu-MuiTypography-root"
              />
            </li>
          </ol>
        </nav>
        <div
          class="MuiStack-root css-1oxswqj-MuiStack-root"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary css-74d805-MuiButtonBase-root-MuiButton-root"
            id=":r0:"
            tabindex="0"
            type="button"
          >
            Sign out
          </button>
        </div>
      </div>
      <h6
        class="MuiTypography-root MuiTypography-h6 css-1miy0lu-MuiTypography-root"
        element="h2"
      >
        Select a Board
      </h6>
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-1o9gf9u-MuiPaper-root-MuiCard-root"
        style="--Paper-shadow: 0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12);"
      >
        <div
          class="MuiCardContent-root css-1lt5qva-MuiCardContent-root"
        >
          <div
            class="MuiTypography-root MuiTypography-h5 css-16xl4zq-MuiTypography-root"
          >
            <a
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-18rqtra-MuiTypography-root-MuiLink-root"
              href="/boards/new"
            >
              + Add New Board
            </a>
          </div>
        </div>
      </div>
    </div>
  </main>
</div>
`;
